From a639701d3f528aea313d0062b9ae863f2660f546 Mon Sep 17 00:00:00 2001
From: Thomas Lamprecht <t.lamprecht@proxmox.com>
Date: Wed, 6 Apr 2016 16:43:38 +0200
Subject: [PATCH 49/52] backup: bdrv_set_enable_write_cache is no more

see: 09cf9db1bcd60d9889b774925ba7058286d35412

this reverts a hunk of our commit: backup: modify job api
---
 block/backup.c | 9 +++------
 1 file changed, 3 insertions(+), 6 deletions(-)

diff --git a/block/backup.c b/block/backup.c
index 8801b47..8baaf36 100644
--- a/block/backup.c
+++ b/block/backup.c
@@ -414,12 +414,9 @@ static void coroutine_fn backup_run(void *opaque)
 
     job->done_bitmap = bitmap_new(end);
 
-    if (target) {
-        bdrv_set_enable_write_cache(target, true);
-        if (target->blk) {
-            blk_set_on_error(target->blk, on_target_error, on_target_error);
-            blk_iostatus_enable(target->blk);
-        }
+    if (target && target->blk) {
+        blk_set_on_error(target->blk, on_target_error, on_target_error);
+        blk_iostatus_enable(target->blk);
     }
 
     bdrv_add_before_write_notifier(bs, &before_write);
-- 
2.1.4

